dev/tools/gen_date_localizations.dart

```diff
@@ -0,0 +1,147 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:args/args.dart' as args;
+import 'package:path/path.dart' as path;
+
+const String _kCommandName = 'gen_date_localizations.dart';
+
+Future<Null> main(List<String> rawArgs) async {
+  final bool writeToFile = (new args.ArgParser()..addFlag('write', abbr: 'w', defaultsTo: false)).parse(rawArgs)['write'];
```

Comment:
It would be good to update gen_localizations.dart to be consistent vis this 'w' flag.

Give the 'w' flag a --overwrite synonym (like dartfmt)?

Likewise for the requirement (enforced below) that the app be run from the isRepoRoot.